### DISCUSSION TITLE
petShopApp

### DISCUSSION THREAD

**Discussion by kaiosatiro** [TomSchimansky/CustomTkinter#2105] [Showcase]
Hey dear friends. Thank you so much. I learned so much from reading the issues here.
I made this app to help a friend manage his customers in his pet shop. I made it to help him and to learn in the process.

I'm still an amateur so, any advice on good practices is welcome. The App is in Portuguese in case someone wonders.

I hope in the future I can contribute a lot more.

https://github.com/kaiosatiro/petshopApp

https://github.com/TomSchimansky/CustomTkinter/assets/87156189/9d6cae62-887e-4cb3-9be7-7cffa2364568

**Comment by avalon60** [comment]
Cool. Also nice to see someone using the right tool for this kind of thing (a relational database/SQL).
